django_project/lesson/urls.py

```diff
@@ -19,25 +19,17 @@
     WorksheetDeleteView,
     WorksheetListView,
 )
+from lesson.views.specification import (
+    SpecificationCreateView,
+    SpecificationListView,
+    SpecificationOrderView,
+    SpecificationOrderSubmitView,
+    SpecificationUpdateView,
+    SpecificationDeleteView
+)
+
 urlpatterns = patterns(
     '',
-    # Worksheet
-    url(regex='^(?P<project_slug>[\w-]+)/section/'
-              '(?P<section_slug>[\w-]+)/worksheet/create/$',
-        view=WorksheetCreateView.as_view(),
-        name='worksheet-create'),
-    url(regex='^(?P<project_slug>[\w-]+)/section/'
-              '(?P<section_slug>[\w-]+)/update/(?P<pk>[\w-]+)/$',
-        view=WorksheetUpdateView.as_view(),
-        name='worksheet-update'),
-    url(regex='^(?P<project_slug>[\w-]+)/section/'
-              '(?P<section_slug>[\w-]+)/delete/(?P<pk>[\w-]+)/$',
-        view=WorksheetDeleteView.as_view(),
-        name='worksheet-delete'),
-    url(regex='^(?P<project_slug>[\w-]+)/section/'
-              '(?P<section_slug>[\w-]+)/worksheet/list/$',
-        view=WorksheetListView.as_view(),
-        name='worksheet-list'),
```

Comment:
ok noted. sorry if its wrong, I thought it will be better if we are putting it in a pattern/order

Reply:
Yes, I understand why you did that. No problem. It's just because I also have edited/updated URLS in `worksheet` so I run into conflicts because you move this code ;-)